graphQl/resolvers.js

```diff
@@ -32,30 +34,29 @@ module.exports = {
       name: UserInput.name,
     });
     const createdUser = await user.save();
-    console.log(createdUser);
     return createdUser;
   },
   async login({ email, password }) {
-    const user = await User.findOne({ email: email });
+    const user = await User.findOne({ email });
     if (!user) {
-      const error = new Error("User does not exist");
+      const error = new Error('User does not exist');
       throw error;
     }
     const isEqual = await bcrypt.compare(password, user.password);
```

Comment:
`passwordsMatch` makes more sense. Use this or think of a variable name that is more descriptive.